applications/Preprocessor/centaur.cpp

```diff
@@ -237,7 +233,9 @@ CentaurReader::CentaurReader(std::string filename) {
     // Interface panels (3D only)
     // 1 data line with number N of interface panels (could be zero)
     // 1 data line with N entries (panel numbers?)
-    skipGroup();
+    if (is3D()) {
+        skipGroup();
+    }
```

Comment:
The comment and the command together are a bit misleading

Reply:
You mean that there are two data lines listed?

Because that is the standard 'group' format (see also point 4 at the top of the file).